utils/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.8
-PKG_RELEASE:=2
+PKG_VERSION:=2017-02-09
```

Comment:
nmav means that users (and  "opkg") should recognize a newer package by comparing the version numbers
1.8 < 2017-02-09
but 2017-02-09 vs. 1.9 ???

Maybe "1.8.git-2017-02-09" to indicate that it is 1.8 plus something.